src/main.rs

```diff
@@ -84,17 +84,17 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
             event: WindowEvent::KeyboardInput { input, .. },
             ..
         } => {
-            if input.virtual_keycode == Some(VirtualKeyCode::LWin) {
+            if input.virtual_keycode == Some(VirtualKeyCode::LControl) {
```

Comment:
I had this as `LWin` as on macOS quit is `Command-Q` and not `Ctrl-Q` (`Command` and `LWin` are the same thing I believe). We should probably add some platform checks to make sure each platform get their correct keybindings.